modules/cluster-logging-upgrading-logging.adoc

```diff
@@ -0,0 +1,154 @@
+
+.. Click the *Elasticsearch Operator*.
+
+.. Click *Subscription* -> *Channel*. 
+
+.. In the Change Subscription Update Channel window, select *4.2* and click `Save`.
+
+.. Wait for a seconds, then click *Operators* -> *Installed Operators*. 
++
+The Elasticsearch Operator is shown a 4.2. For example:
++
+----
+Elasticsearch Operator
+4.2.0-201909201915 provided 
+by Red Hat, Inc
+----
+
+. Upgrade the Cluster Logging Operator:
+
+.. From the web console, click *Operators* -> *Installed Operators*. 
+
+.. Select the `openshift-logging` Project.
+
+.. Click the *Cluster Logging Operator*.
+
+.. Click *Subscription* -> *Channel*. 
+
+.. In the Change Subscription Update Channel window, select *4.2* and click `Save`.
+
+.. Wait for a seconds, then click *Operators* -> *Installed Operators*. 
++
+The Cluster Logging Operator is shown a 4.2. For example:
++
+----
+Cluster Logging
+4.2.0-201909201915 provided 
+by Red Hat, Inc
+----
+
+. Verify the upgrade by checking the logging component pods.
+
+.. Ensure that the Elasticsearch pods are using the 4.2 image:
++
+----
+$ oc get pod -o yaml -n openshift-logging --selector component=elasticsearch |grep 'image:'
+
+image: registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.2.0-201909201915
+----
++
+.. Ensure that all Elasticsearch pods are in the *Ready* status:
++
+----
+$ oc get pod -n openshift-logging --selector component=elasticsearch
+
+NAME                                            READY   STATUS    RESTARTS   AGE
+elasticsearch-cdm-1pbrl44l-1-55b7546f4c-mshhk   2/2     Running   0          31m
+elasticsearch-cdm-1pbrl44l-2-5c6d87589f-gx5hk   2/2     Running   0          30m
+elasticsearch-cdm-1pbrl44l-3-88df5d47-m45jc     2/2     Running   0          29m
+----
++
+.. Ensure that the Elasticsearch cluster is healthy:
++
+----
+oc exec -n openshift-logging -c elasticsearch elasticsearch-cdm-1pbrl44l-1-55b7546f4c-mshhk -- es_cluster_health
+
+{
+  "cluster_name" : "elasticsearch",
+  "status" : "green",
+
+....
+
+----
+
+.. Ensure that the logging collector pods are using to 4.2 images and pods status are ready:
```

Comment:
I don't see anything in the below output for "ready", is this correct to say here?

If so, the wording is a little off (and a "to" typo). Maybe

```
Ensure that the logging collector pods are using 4.2 images and are ready:
```